proto/evm/v1beta1/genesis.proto

```diff
@@ -25,6 +25,8 @@ message GenesisState {
 
     Gateway gateway = 9 [ (gogoproto.nullable) = false ];
     repeated ERC20TokenMetadata tokens = 10 [ (gogoproto.nullable) = false ];
+    bytes latest_batched_commands_id = 11
```

Comment:
Can't this be calculated with `command_batches`?

Reply:
Batches do not have a timestamp we can use to sort them, and I don't think we should rely on the order that we get from the genesis file (I don't think the iterator we use during export guarantees FIFO order).

But I guess one way to calculate the latest batch would be to just iterate through all of them until we find an ID that is not referenced in the `prev_batched_commands_id` field of any of the batches. Does this makes sense to you?

Reply:
Added more checks during validation of genesis state to make sure each batch references the previous one in the `prev_batched_commands_id ` field

Reply:
Yes `prev_batched_commands_id` is exactly what can be and should be used to figure out order.